app/models/location.rb

```diff
@@ -361,9 +361,15 @@ class Location < ApplicationRecord
 
   def display_attributes
     [
-      { name: "Location code ", value: location_code, suffix: false },
-      { name: "Postcode", value: postcode, suffix: county },
-      { name: "Type of unit", value: type_of_unit, suffix: false },
+      { name: "Postcode", value: postcode },
+      { name: "Local authority", value: location_admin_district },
+      { name: "Location name", value: name, edit: true  },
+      { name: "Total number of units at this location", value: units },
+      { name: "Common type of unit", value: type_of_unit },
+      { name: "Mobility type", value: mobility_type },
+      { name: "Code", value: location_code },
+      { name: "Availability", value: "Available from #{available_from.to_formatted_s(:govuk_date)}" },
+      { name: "Status", value: status },
     ]
   end
```

Comment:
I don't think these values belong on the model as they are specifically for views. Consider moving into a helper for locations?

Reply:
Is it better now?